packages/paste-website/src/pages/components/tabs/index.mdx

```diff
@@ -190,6 +214,33 @@ using the `select` function from the tab state to advance to the next tab, and g
   {StateHookTabsExample}
 </LivePreview>
 
+## Composition notes
+
+Tabs should never contain interactive elements, as doing so makes it difficult for users of assistive technologies to focus or click on the Tabs. Interactive elements may live within the Tab Panels.
+
+Tab labels should be relatively short to improve readability. Users should be able to easily scan the Tabs and know what type of content to expect. Try to keep labels to 1-2 words each.
+
+Labels should be parallel. For example, all nouns (Documents, Images, Downloads).
```

Comment:
```suggestion
Labels should be parallel parts of speech. For example, all nouns (Documents, Images, Downloads).
```

Being specific here because "parallel" by itself could be interpreted as visual design guidance.